protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -51,12 +54,20 @@ contract RamosStrategy  is AbstractStrategy, IRamosProtocolTokenVault {
         templeToken.mint(recipient, amount);
     }
 
+    function borrowQuoteToken(uint256 amount, address recipient) external {
+        quoteToken.safeTransfer(recipient, amount);
+    }
+
     function repayProtocolToken(uint256 amount) external onlyElevatedAccess {
         // @todo This should be repaid to the TRV, so the dTemple can be burned
         templeToken.safeTransferFrom(msg.sender, address(this), amount);
         templeToken.burn(amount);
     }
 
+    function repayQuoteToken(uint256 amount) external {
+        quoteToken.safeTransferFrom(msg.sender, address(this), amount);
```

Comment:
For the v2 strategy, it should be pulling the funds from the sender (as you have here) and then repaying the funds to the TRV

`treasuryReservesVault.repay()`

Reply:
gotcha, I was missing this part on my mind